CHANGELOG.md

```diff
@@ -3,7 +3,64 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.30.0
+
+This release contains a number of enhancements and fixes.
+
+### Server and Runtime
+
+- Support listening on abstract Unix Domain Sockets ([#3533](https://github.com/open-policy-agent/opa/issues/3533)) authored by @[amanymous-net](https://github.com/amanymous-net)
+- Support minimum TLS version configuration, default to 1.2 ([#3226](https://github.com/open-policy-agent/opa/issues/3226)) authored by @[kale-amruta](https://github.com/kale-amruta)
+- Bugfix: treat missing/empty roots as owning all paths ([#3521](https://github.com/open-policy-agent/opa/issues/3521))
+- plugins/rest: Add option to specify CA for remote services ([#1954](https://github.com/open-policy-agent/opa/issues/1954))
+- Extend POST v1/query endpoint to accept input, refactor index.html to use fetch()
+- Update downloader's Etag to value of the last successful activation
```

Comment:
Nit: I'd add something like "In case of bundle download or activation errors, reset the cached etag to the last successful activation. Previously OPA would reset the cached etag entirely (which could trigger unnecessary bundle downloads in edge-case scenarios.)"